### DISCUSSION TITLE
Feedback for the landing page

### DISCUSSION THREAD

**Discussion by Andre601** [squidfunk/mkdocs-material#3959] [General]
First of, I like the design of the landing page and stuff such as the animations and transitions used.
With that said, are there a few minor changes that should be made, as I feel like it could help with readability (and maybe accessibility?).

The first thing I would change is the background behind the "Documentation that simply works".
It feels kinda hard right now to read it with the background image. I can't directly point to what exactly makes it so difficult for me, but maybe have a gradient there could help?

Next, while I don't mind it too much, is the usage of this bright white background for the detailed descriptions (Built-in search, Code annotations, etc) not that good on the eyes. Especially if you are someone like me who prefers dark-themed pages for easier reading and less pain on the eyes.
It's even more weird considering you prominently display the theme toggle at the top, but it has no effect on the landing page itself.

That's already everything I have to give feedback for. I hope you see this as possible ways to improve the documentation further in the future.
You're also free to politely ignore this.

**Comment by squidfunk** [comment]
Thanks for taking the time to give feedback. Some valid points indeed. I already have some ideas, give me some time 😊
